app/backplane/radio_module/src/c_lora_transmit_tenant.cpp

```diff
@@ -0,0 +1,35 @@
+#include "c_lora_transmit_tenant.h"
+#include "c_radio_module.h"
+
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(CLoraTransmitTenant);
+
+void CLoraTransmitTenant::Startup() {
+}
+
+void CLoraTransmitTenant::PostStartup() {
+}
+
+void CLoraTransmitTenant::Run() {
```

Comment:
same idea as lora to udp tenant. If this isn;t the only tenant for a task the second tenant will get starved since this loop will never break